src/account.ts

```diff
@@ -192,3 +284,21 @@ export const importPublic = function(publicKey: Buffer): Buffer {
   }
   return publicKey
 }
+
+/**
+ * Returns a zero address.
+ */
+export const zeroAddress = function(): string {
+  const addressLength = 20
+  const addr = zeros(addressLength)
+  return bufferToHex(addr)
+}
+
+/**
+ * Checks if a given address is a zero address.
+ */
+export const isZeroAddress = function(hexAddress: string): boolean {
+  assertIsHexString(hexAddress)
```

Comment:
Can we explicitly check that this `hexAddress` has the expected length?

Reply:
that function wasn't introduced in this pr, i just moved it to the bottom of the file to organize.

it does check the length though in its string comparison with `zeroAddress()`?

Reply:
It does, but I do not think it is right if you input a string like "0x00" - this is not a zero address, but it is not a valid address anyways because it is too short. I'd say that raising an error here is better for the end user.

Reply:
yeah well that would return `false` which is correct, right?

i don't mind the idea of adding an error based on length, but technically that would be a breaking change since people with its current use may not expect an error to be thrown. so i'll leave it for now, but if you feel strongly, we can add in a separate pr.